src/smokeTest/java/uk/gov/hmcts/reform/hmi/people/HealthCheckSmokeTest.java

```diff
@@ -15,6 +16,7 @@ void setup() {
     }
 
     @Test
+    @Disabled
     void peopleHealthCheckTest() {
```

Comment:
Is this test correct to be disabled and if so could we add a comment detailing why?